packages/@aws-cdk/aws-iam/test/role.test.ts

```diff
@@ -235,6 +235,49 @@ describe('IAM role', () => {
     });
   });
 
+  describe('role path', () => {
+    test('can be used to specify the path', () => {
+      const stack = new Stack();
+
+      new Role(stack, 'MyRole', { path: '/', assumedBy: new ServicePrincipal('sns.amazonaws.com') });
+
+      expect(stack).toHaveResource('AWS::IAM::Role', {
+        Path: '/',
+      });
+    });
+
+    test('must be between 1 chars and 512 chars', () => {
+      const stack = new Stack();
+
+      const assumedBy = new ServicePrincipal('bla');
+
+      new Role(stack, 'MyRole1', { assumedBy, path: '/' });
+      new Role(stack, 'MyRole2', { assumedBy, path: '/p/a/t/h/' });
+
+      const expected = (val: any) => `role path is set to ${val}, but the length must be >= 1 and <= 512.`;
+      expect(() => new Role(stack, 'MyRole3', { assumedBy, path: '' }))
+        .toThrow(expected(''));
+      expect(() => new Role(stack, 'MyRole4', { assumedBy, path: '/' + Array(512).join('a') + '/' }))
+        .toThrow(expected('/' + Array(512).join('a') + '/'));
+    });
+
+    test('must match regular expression', () => {
```

Comment:
Same comment as the test above. One test case per test, please.

Reply:
Fixed in ea3a32439144a1f3597dd0bf14e41f3fe16973b9